build.gradle

```diff
@@ -54,6 +54,7 @@ ext {
     supportV4CoreUi = "com.android.support:support-core-ui:$supportLibraryVersion"
     supportV4Compat = "com.android.support:support-compat:$supportLibraryVersion"
     supportRecyclerView = "com.android.support:recyclerview-v7:${supportLibraryVersion}"
+    supportCardView = "com.android.support:cardview-v7:${supportLibraryVersion}"
```

Comment:
Is it okay to add the `cardview` dependency? I found no other good looking way to add elevation to the new popup, while maintaining backwards compability.